models/wrf/tutorial/README.rst

```diff
@@ -612,217 +612,64 @@ directory ``output/2017042700/PRIORS`` with names like *prior_d01.0001*,
-         lon1       = 270.0,
-         lon2       = 330.0
-         /
-
-   Choosing "select_obs = 0" will select all the observations in the
-   ASCII file. Set "ObsBase" to the directory you output the files from
-   during the last step. If you wish to choose specific observations
-   from the ASCII intermediate file or control other program behavior,
-   there are many namelist options documented on the
-   `create_real_obs <../../../observations/obs_converters/NCEP/ascii_to_obs/create_real_obs.html>`__
-   page.
-
--  It is now time to build *ascii_to_obs* programs. Run the following:
-
-   ::
-
-      cd $DART_DIR/observations/obs_converters/NCEP/ascii_to_obs/work
-      ./quickbuild.sh
-
--  Run the *create_real_obs* program to create the DART observation
-   sequence files:
-
-   ::
 
-      cd $DART_DIR/observations/obs_converters/NCEP/ascii_to_obs/work
-      ./create_real_obs
-
--  The program *create_real_obs* will create observation sequence files
-   with one file for each six hour window. For a cycled experiment, the
-   typical approach is to put a single set of observations, associated
-   with a single analysis step, into a separate directory. For example,
-   within the ``output`` directory, we would create directories like
-   ``2017042700``, ``2017042706``, ``2017042712``, etc. for 6-hourly
-   cycling. Place the observation files in the appropriate directory to
-   match the contents in the files (e.g. *obs_seq2017042706*) and rename
-   as simply *obs_seq.out* (e.g. ``output/2017042706/obs_seq.out``).
-
--  It is helpful to also run the
-   `wrf_dart_obs_preprocess <../../../models/wrf/WRF_DART_utilities/wrf_dart_obs_preprocess.html>`__
-   program, which can strip away observations not in the model domain,
-   perform superobservations of dense observations, increase observation
-   errors near the lateral boundaries, check for surface observations
-   far from the model terrain height, and other helpful pre-processing
-   steps. These collectively improve system performance and simplify
-   interpreting the observation space diagnostics. There are a number of
-   namelist options to consider, and you must provide a *wrfinput* file
-   for the program to access the analysis domain information.
-
-Step 4: Overview of forward (observation) operators [OPTIONAL] 
+Step 4: Overview of forward operators [Informational Only] 
 --------------------------------------------------------------
 
 This section is for informational purposes only and does not include any 
-instructions to complete the tutorial. It provides a description of
+required steps to complete the tutorial. It provides a description of
 the DART settings that control the forward operator which
 calculates the prior and posterior model estimates for the observations. 
 An introduction to important namelist variables that control the operation of the forward
 operator are located in the `WRF namelist documentation. 
 <../../../models/wrf/readme.html#namelist>`__
```

Comment:
:doc: or :ref: rather than html

Reply:
How do you do a :doc: ref when you want to refer to a header like #namelist, not just the page?

Reply:
You can add an [anchor](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-a-custom-anchor) in wrf/readme.rst

e.g.

.. _wrfnamelist: